packages/react-ui-core/src/ModalStack/ModalStack.js

```diff
@@ -109,10 +114,7 @@ export default class ModalStack extends PureComponent {
 
   toggleBodyClass(toggle) {
     const { theme } = this.props
-
-    if (this.modalHost) {
-      document.body.classList.toggle(theme['Overlay-lock'], toggle)
-    }
+    document.body.classList.toggle(theme['Overlay-lock'], toggle)
```

Comment:
I'm curious if this works as expected in IE - according to `toggle()` docs IE does not support second argument: https://developer.mozilla.org/en-US/docs/Web/API/Element/classList